dvjsharma/Drawn2Shoe
# Add Logout Button and its functionality

## Description
This fixes #30.
I have added a logout button which is only activated once a user has logged in. When the logout button is clicked, it logs the user out.

### Changes

I have added a Logout.jsx page for the front end aspects and also added necessary parts to App.jsx. 
I have made some changes to navbar.jsx to make it more readable and also added the logout functionality. 
I have made necessary changes to the backend server side codes of controller/user.js and router/user.js

## How to test

Just log in using any user name and password.
![image](https://github.com/dvjsharma/Drawn2Shoe/assets/111494403/bf05453f-3d2e-42c4-a792-5d0e98c16b93)

You will notice the logout button appears next to the name.
![image](https://github.com/dvjsharma/Drawn2Shoe/assets/111494403/8e6c8c88-0c57-4f95-977b-8e4b54dc625b)

Click the logout button to be successfully logged out. Log out button disappears after this.
https://github.com/dvjsharma/Drawn2Shoe/assets/111494403/eb1d48ae-1451-4876-b4a0-152aa072f3a4

Resolves #30
